src/OpenTracing/Tag.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace OpenTracing;
+
+class Tag
+{
+
+    const PEER_SERVICE = "peer.service";
+    const PEER_HOSTNAME = "peer.hostname";
+    const PEER_HOST_IPV4 = "peer.ipv4";
+    const PEER_HOST_IPV6 = "peer.ipv6";
+    const PEER_PORT = "peer.port";
```

Comment:
PHP 5 can assure the argument is an instance of a given class (or its subclass)
PHP 7 can assure the scalar types for arguments too (it's released for a few months now so we cannot aim at this version yet)